corehq/apps/analytics/static/analytix/js/gtx.js

```diff
@@ -63,15 +61,58 @@ hqDefine('analytix/js/gtx', [
         });
     };
 
+    /**
+     * Modified version of utils.initApi to enable GTM on the India environment.
+     * This does not checks for `global.isEnabled` and instead directly checks for the environment.
+     * This is done to avoid enabling other analytics tooling on the India environment.
+     * See the PR description (https://github.com/dimagi/commcare-hq/pull/35238) for more details.
+    */
+    var initApi = function (ready, apiId, isGTMEnabled, scriptUrls, logger, initCallback) {
+        logger.verbose.log(apiId || "NOT SET", ["DATA", "API ID"]);
+
+        if (_.isString(scriptUrls)) {
+            scriptUrls = [scriptUrls];
+        }
+
+        if (!isGTMEnabled) {
```

Comment:
Though I would ask to add a comment here in file to highlight this one line that is different from the rest of the code.

I wonder if its possible to still use the original function by adding an argument like `checkForGTMEnabled` that modifies the check you need instead of copying the function and modifying one line?
Feel free to punt it if its not worth the effort and if this is just temporary.